app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/AccessApiFragment.kt

```diff
@@ -80,47 +81,46 @@ class AccessApiFragment : Fragment() {
                 is GetAccountResult.NoAccountFound -> {
                     displaySignedOutState()
                 }
+                is GetAccountError -> {
+                    displayDialog(getString(R.string.msal_exception_title), accountResult.exception?.message ?: accountResult.errorMessage)
```

Comment:
For simplicity, let's just use `accountResult.errorMessage` here. That's the ultimate error. For more details on the error, developers can look into the exception's message. What do you think? 
If you agree, this comment applies to multiple places.

Reply:
Agreed. Will make the update